client/my-sites/plugins/plugins-browser/index.jsx

```diff
@@ -557,6 +558,11 @@ const PluginBrowserContent = ( props ) => {
 		<>
 			{ ! props.jetpackNonAtomic ? (
 				<>
+					<Spotlight
+						taglineText="Drive more traffic with Yoast SEO"
+						illustrationSrc="https://ps.w.org/wordpress-seo/assets/icon-256x256.png?rev=1550389"
+						url={ `/plugins/wordpress-seo-premium/${ props.siteSlug }` }
+					/>
```

Comment:
Looks good, could we move this to the featured component? i.e. just above this line, https://github.com/Automattic/wp-calypso/blob/ed4498ba45e51db846c7e11f231e1576089ec931/client/my-sites/plugins/plugins-browser-list/index.jsx#L97 with something like:

```
{ listName === 'featured' && <Spotlight ... /> }
```